tests/integration/project_test.py

```diff
@@ -174,6 +174,21 @@ def test_project_up(self):
         project.kill()
         project.remove_stopped()
 
+    def test_project_up_starts_uncreated_services(self):
+        db = self.create_service('db')
+        web = self.create_service('web', links=[(db, 'db')])
+        project = Project('composetest', [db, web], self.client)
+        project.up(['db'])
+        self.assertEqual(len(project.containers()), 1)
+
+        project.up()
+        self.assertEqual(len(project.containers()), 2)
+        self.assertEqual(len(db.containers()), 1)
+        self.assertEqual(len(web.containers()), 1)
+
+        project.kill()
+        project.remove_stopped()
```

Comment:
Are they needed at all, actually? The `setUp` method of `DockerClientTestCase` takes care of cleanup.

It'd be good to clean up *after* a test run, but we've currently got loads of testcases that don't do that.